### DISCUSSION TITLE
咨询下关于图表类元素的动画能否使用libpag实现或者有别的思路吗，可以设计师提供文件，开发做数据和坐标轴配置

### DISCUSSION THREAD

**Discussion by Jiiiin** [Tencent/libpag#1700] [Q&A] [answered]
https://github.com/Tencent/libpag/assets/22162992/17e359c4-69a4-4cea-a382-4e9decbd2c43

**Comment by kevingpqi123** [comment, marked as answer]
这类功能需要开发者通过代码实现，pag 方案是还原 AE 效果，更多的是一种动画模版，AE 中动画基本上都是确定的动画轨迹，不太适合这种场景

**Reply by Jiiiin** [reply]
好的感谢
